include/mg_procedure.h

```diff
@@ -1290,6 +1290,9 @@ struct mgp_module;
 /// Describes a procedure of a query module.
 struct mgp_proc;
 
+/// Describes a Memgraph magic function.
+struct mgp_func;
+
```

Comment:
Since there are no write capabilities in functions, and vertex/edge has an instance on "non-writable" mgp_graph, do I actually need this in the API? Pretty sure I don't :sweat_smile: 

@antaljanosbenjamin